tools/spelling_dictionary.txt

```diff
@@ -337,6 +337,7 @@ XNOR
 XSS
 YAML
 ZXID
+aaa
```

Comment:
@zuercher we really need to fix this. If you are up for a tech debt item can you potentially add some type of escape hatch to the spell checker that turns it off and on like we have for other format things? I think there is an issue tracking this.

Reply:
Would it be feasible to skip the spell check for text inside double back ticks?

Reply:
Yes I'm going to work on that next.